client/my-sites/domains/domain-management/dataviews/use-fields.tsx

```diff
@@ -0,0 +1,152 @@
+import { LoadingPlaceholder } from '@automattic/components';
+import { PartialDomainData } from '@automattic/data-stores';
+import { DomainsTableExpiresRenewsOnCell } from '@automattic/domains-table/src/domains-table/domains-table-expires-renews-cell';
+import { DomainsTableSiteCell } from '@automattic/domains-table/src/domains-table/domains-table-site-cell';
+import { Field } from '@wordpress/dataviews';
+import { useTranslate } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { DomainField } from './dataviews-fields/domain-field';
+import { DomainStatusField } from './dataviews-fields/domain-status-field';
+import { SslStatusField } from './dataviews-fields/ssl-status-field';
+import { useDomainsDataViewsContext } from './use-context';
+
+interface Props {
+	openDomainPane?: ( domain: PartialDomainData ) => void;
+}
+
+export function useFields( { openDomainPane }: Props ) {
+	const translate = useTranslate();
+	const {
+		isAllSitesView,
+		domainsRequiringAttention,
+		sites,
+		isLoadingSites,
+		selectedFeature,
+		getSiteSlug,
+		getFullDomain,
+		hasConnectableSites,
+	} = useDomainsDataViewsContext();
+
+	const fields = useMemo< Field< PartialDomainData >[] >(
+		() => [
+			{
+				id: 'domain_name',
+				label: translate( 'Domains' ),
+				enableHiding: false,
+				enableSorting: true,
+				enableGlobalSearch: true,
+				getValue: ( { item }: { item: PartialDomainData } ) => item.domain,
+				render: ( { item }: { item: PartialDomainData } ) => (
+					<DomainField
+						domain={ item }
+						isAllSitesView={ isAllSitesView }
+						selectedFeature={ selectedFeature }
+						openDomainPane={ openDomainPane }
+					/>
+				),
+			},
+			{
+				id: 'owner',
+				label: translate( 'Owner' ),
+				enableHiding: false,
+				enableSorting: true,
+				getValue: ( { item }: { item: PartialDomainData } ) => {
+					const domain = getFullDomain( item );
+					if ( ! domain ) {
+						return '';
+					}
+					// Removes the username that appears in parentheses after the owner's name.
+					// Uses $ and the negative lookahead assertion (?!.*\() to ensure we only match the very last parenthetical.
+					return domain.owner.replace( / \((?!.*\().+\)$/, '' );
```

Comment:
The render function checks that `domain.owner` exists but this doesn't. Which one is correct? Is this missing a check or the check is redundant in the render?

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/97677/commits/91bececa44af9c57eda117e22f2c27510f60e08d